routes/botSettings.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const botSettings = require('../db/botSettings.js')
+const router = express.Router()
+
+module.exports = router;
+// reset endpoint, use only in testing
+router.get('/start', async (req, res) => {
+    await botSettings.destroy({where: {}})
+    let bot = await botSettings.build({'id': 1,
+                    'botToken': 'Token', 
+                    'emojies':
+                        {'like':'👍',
+                        'dislike':'👎',
+                        'message':'📝',
+                        'report':'‼️'}})
+    bot.save()
+    res.send(200)
+})
+// update data endpoint
+router.put('/change', async (req, res) => {
```

Comment:
"update" is a better word than "change" for such tasks. If you only update bot token, then call this "update-token" to be more declarative.